extension/goplugin/environment.go

```diff
@@ -144,6 +146,14 @@ func (thisEnvironment *Environment) Database() goext.IDatabase {
 	return thisEnvironment.extDatabase
 }
 
+func (thisEnvironment *Environment) SetEnvBindings() {
```

Comment:
this function is not Setting, it's Reseting. I'd go for "ResetEnvBindings", or simply "Rebind"